include/bpf2c.h

```diff
@@ -20,6 +21,7 @@ typedef unsigned long long uint64_t;
 #include <stdbool.h>
 #include <stddef.h>
 #include <stdint.h>
+#include <intrin.h>
```

Comment:
Move include up into alphabetical order per [DevelopmentGuide](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#header-files).

Reply:
Fixed.